### PR TITLE
Add solicitor import type

### PR DESCRIPTION
**Pull request by NickDelfino**
This is for [jira task](https://evertroops.atlassian.net/browse/ET-8792).

This creates a new recognized import type called solicitors. This is for relationship management.

### CODE REVIEW

File: files/default/process_uploads [hunk 1]
```diff
@@ -274,7 +278,9 @@ def process(org_slug, path, compression)
   assignment_import = !(File.basename(path) =~ /\.assignments?\./i).nil?
   logger.debug "Assignments import? #{assignment_import}"
 
-  
+  solicitor_import = !(File.basename(path) =~ /\.solicitors?\./).nil?
```
Comment (grosezoom) [inline files/default/process_uploads]: you're missing the "I" in ./i).nil.  Is that important?

File: files/default/process_uploads [hunk 2]
```diff
@@ -167,6 +169,8 @@ def post_to_importer(oid, s3_filename, compression, full_import, gift_import, in
     data['type'] = 'PROPOSAL_CSV'
   elsif assignment_import
     data['type'] = 'ASSIGNMENT_CSV'
+  elsif solicitor_import
+    data['type'] = 'SOLICITOR_CSV'
```
Comment (grosezoom) [inline files/default/process_uploads]: Same as above

File: files/default/process_uploads [hunk 3]
```diff
@@ -141,6 +141,8 @@ def get_header_change_status_from_importer(oid, s3_filename, compression, gift_i
...
```
Comment (grosezoom) [inline files/default/process_uploads]: Is type matching to the import type?  If so, it should be ASSIGNMENT_TEAM_CSV

File: files/default/process_uploads (hunk 1 above)
Comment (NickDelfino) [reply files/default/process_uploads]: updated!

File: files/default/process_uploads (hunk 2 above)
Comment (NickDelfino) [reply files/default/process_uploads]: updated!

File: files/default/process_uploads (hunk 3 above)
Comment (NickDelfino) [reply files/default/process_uploads]: updated!

**Pull request merged**